cmd/state-installer/cmd.go

```diff
@@ -250,6 +250,16 @@ func execute(out output.Outputer, cfg *config.Instance, an analytics.Dispatcher,
 		return errs.Wrap(err, "Could not detect if State Tool is already installed.")
 	}
 
+	if !stateToolInstalled && fileutils.DirExists(params.path) {
```

Comment:
Please add a comment above, the installer in particular is very much geared towards outsiders reading this code.